charts/athens-proxy/values.yaml

```diff
@@ -94,6 +94,15 @@ storage:
     # as GCP figures out internal authentication between products for you.
     serviceAccount: ""
 
+# Container security context configuration
+# This will override the `image.runAsNonRoot` settings in the specified container if `runAsUser` or `runAsGroup` are set
+securityContext: {}
```

Comment:
Where is this being used?

Reply:
accidentally used the `initContainerSecurityContext` twice, I have updated the template